bb/ramfs.go

```diff
@@ -88,46 +110,64 @@ func ramfs() {
 		log.Fatalf("%v\n", err)
 	}
 
-	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = "init"
+	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = []string{"init", "ubin"}
 
-	// For now, just one added path.
 	if *extraPaths != "" {
-		x := strings.Split(*extraPaths, ":")
-		if len(x) != 2 {
-			log.Fatalf("extraPaths requires two : separated pathnames")
+		extras := strings.Split(*extraPaths, " ")
+		for _, x := range extras {
+			paths["/"] = append(paths["/"], x)
```

Comment:
So what exactly is the format here? Say I have two extra paths, do I do 
/:etc/hosts /:lib/modules/4.04
?